src/extensions/mountainview/MVSortingView/PreloadCheck.tsx

```diff
@@ -98,18 +99,23 @@ const PreloadCheck: FunctionComponent<Props> = ({ recording, sorting, children }
 
     return (
         <Fragment>
-            <Dialog
-                open={status !== 'finished'}
-            >
-                {error ? <Status message={`Error: ${error.message}`} /> : <Status message={message} />}
-            </Dialog>
+            <BlockInteraction block={status !== 'finished'} {...{width, height}} message={error ? `Error: ${error.message}` : message} />
```

Comment:
Now we just block interaction to the GUI component and not the entire browser window. This is especially important for jupyter notebook (we don't want to block interaction to the entire notebook)